psi4/driver/molutil.py

```diff
@@ -311,7 +249,9 @@ def geometry(geom, name="default"):
         try:
             import pylibefp
         except ImportError as e:  # py36 ModuleNotFoundError
```

Comment:
Do we need to also worry about `ModuleNotFoundError` here?  I suspect not, because I think it derives from `ImportError`, but just want to check that this is the correct way for us to support py3.6+

Reply:
I'm pretty sure this is the right way to support 2.7+. then when we switch to 3.6+ can replace with the more specific `ModuleNotFoundError`